samples/cpp/tutorial_code/introduction/display_image/display_image.cpp

```diff
@@ -6,46 +6,32 @@
 #include <iostream>
 //! [includes]
 
-//! [namespace]
-using namespace cv;
-using namespace std;
```

Comment:
Lets put these under "includes" section or at least "cv". We are trying to avoid using of "cv::" in OpenCV samples code.